cachi2/core/package_managers/rpm/main.py

```diff
@@ -195,6 +195,17 @@ def _download(lockfile: RedhatRpmsLock, output_dir: Path) -> dict[Path, Any]:
             }
             Path.mkdir(dest.parent, parents=True, exist_ok=True)
 
+        for md in arch.metadata:
+            filename = Path(md.url).name.split("-", 2)[1]
```

Comment:
Is this required? Processing the filename would require the URL to point to a file that has the `<checksum>-modules.yaml.gz` format, which is not required. But createrepo might be fine even with the checksum being included.

Reply:
Checked it again and it doesn't needed to remove the checksum.  While I can't remember what happened with previous tests.